lib/AbstractEnvironment.js

```diff
@@ -77,6 +77,29 @@ class AbstractEnvironment {
   removeErrback(errback) {
     this._errbacks = this._errbacks.filter(e => e !== errback);
   }
+
+  /**
+   * Inject libs which are required for parse process
+   *
+   * @protected
+   */
+  _injectVendors() {
+    debug('.inject()-ing parser libs');
+    const filePaths = [
+      require.resolve('../vendor/sizzle.min.js'),
+      require.resolve('../vendor/xhr.sniffer.js'),
+    ];
+    return this._injectFiles(filePaths);
+  }
+
+  /**
+   * @abstract
+   * @param {Array.<string>} filePaths
+   * @protected
+   */
+  _injectFiles(filePaths) {
+    throw new Error('You must redefine _injectFiles method in child environment');
```

Comment:
What's about env which does not support any file injection?
JsDom for example

Reply:
These envs will not invoke this function. Easy, right? ;)

Reply:
You're killing me by your intelligence. You're right!

Reply:
🐗